### ISSUE TITLE
Aurora Alpha Fuel Air Bomb detonation does not grant veterancy

### ISSUE THREAD

**Issue by Generalcamo** [TheSuperHackers/GeneralsGamePatch#283]
As the title says, while the bombs themselves grant veterancy, the resulting explosion does not.

**Comment by xezon** [comment]
Given the powerful destruction of the Alpha Aurora, I assume it would level up quite quickly. So this likely is a controversial change. On the other Hand this makes no sense. Maybe we can fix it and then nerf the veterancy requirements to balance it out.

**Comment by Madrageeeeee** [comment]
Yes think you also need to consider the fact that auroras often get shot down on the way back. If they vet up even quicker they will also give considerable xp to the enemy. 

Think the solution of xezon makes sense in that way.

on the other hand dont think there have ever been issues with this and dont think it is really a necessary adjustment at the moment.

on this topic: think there are also different generals powers that do / do not give xp. Unsure which ones but think a10 does not give xp for things it destroys? Might be worth a separate topic

**Comment by commy2** [comment]
^ A10 and Artillery Strike do not give XP to the player. Other powers do.

**Comment by Jundiyy** [comment]
Vet fix could work but would still result in different outcomes. Currently you kill one Technical and blow another 9 away, no vet up.
You could now increase the amount needed to vet up but the same situation, kill one Technical and blow 9 away, you'd prob vet up.

It's not exactly the same but other units do have a similar design, Infernos with firestorms don't give vet, same for Migs, Dragon Tanks with firewall etc. If they original does not get the kill then the extra won't give vet.

**Comment by RisingZH** [comment]
I think they all should be fixed, if a unit kills another unit/building it should get vet. Flame wall giving vet would be the most impactful. 

Am I right in thinking that you also don’t get vet for killing units which are inside units? (Eg a quad would get the same vet for killing a full Humvee as it would an empty one)

**Comment by commy2** [comment]
> Am I right in thinking that you also don’t get vet for killing units which are inside units?

Yes. If you kill a Humvee, you get the XP awarded for the Humvee only, not the passengers. This is because the passengers die by the Humvee and only indirectly by whatever killed the Humvee.

**Comment by RisingZH** [comment]
Is it possible to change that?

It’s one of the reasons USA is op.

**Comment by commy2** [comment]
I don't know an easy solution to this, sadly.

**Comment by xezon** [comment]
I created a separate issue for that
* TheSuperHackers/GeneralsGameCode#55

**Comment by Madrageeeeee** [comment]
I think this is a very major change to the game. 

Vetted flamers can truly be a gamechanger and if you allow for vetting with flamewalls it will boost tank significantly vs stealth and china vs gla in general. 

So this shudnt be done lightly. 

Flamer spam as tank is already very difficukt to beat as stealth (see latest game of me vs size as an example)

you could also call it an opportunity to excel. For example targeting the tunnel directly before it dies so the flamer gets vetted up. 

With regards to the vet given for units inside a humvee i can sort of see the point and it might actually balance things. 

But here too there is a tradeoff. Unloading ur humvee allows you to use the MDs to laserlock etc but you may also give away easy xp.

hawky for example rarely unloads his vees and there is something to be said for it.
